roles/ansible_tower/tasks/main.yaml

```diff
@@ -30,3 +30,13 @@
 
 - name: Set (httpd_can_network_connect_db) flag on and keep it persistent across reboots
   seboolean: name=httpd_can_network_connect_db state=yes persistent=yes
+
+- name: Setup proot to allow access to /etc/tower/
+  lineinfile:
+    dest: /etc/tower/settings.py
+    backrefs: yes
+    regexp: "{{ item.regex }}"
+    line: '\1\2= {{ item.value }}'
```

Comment:
So, with this, does the end user calling into the role have to know that their regex has to have 2 captures in it for the left side, and that both of those captures have to be for the option name and white space?